packages/odate/odate.0.7/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "hugo.heuzard@gmail.com"
+authors: [ "Hugo Heuzard" ]
+license: "MIT"
+homepage: "https://github.com/hhugo/odate"
+bug-reports:  "https://github.com/hhugo/odate/issues"
+dev-repo: "git+https://github.com/hhugo/odate.git"
+
+synopsis: "Date & Duration Library"
+
+description: """
+Simple date and duration manipulation. Also implement duration printer
+based on string format. Already implemented in opalang
+[http://opalang.org/]. For documentation about the format, see :
+[http://doc.opalang.org/value/stdlib.core.date/Duration/try_generate_printer].
+"""
+
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

Reply:
I'll have dune generate the opam file in the future

Reply:
@haochenx, Are you sure about this change ? It seems like dune is generating `{pinned}` and not `{dev}`

Reply:
ok